src/client/graphql/fragments/CampaignDataFragment.graphql

```diff
@@ -0,0 +1,25 @@
+fragment CampaignData on Campaign {
```

Comment:
nit: should we call these `CampaignDataFragment` to avoid confusing them with GraphQL types?

Reply:
🤔 the problem is that `codegen` will add the word 'Fragment' in the fragments we define by [default](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/data/graphql.tsx#L12246). So I've chosen not to include it here to avoid the creation of a `CampaignDataFragmentFragment` type.

Reply:
haha I don't see the concern about being extra explicit but it's not a big concern